SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -1726,4 +1737,47 @@ bool ActorSystem::EntityHasComponent(const Worker_EntityId EntityId, const Worke
 	}
 	return false;
 }
+
+void ActorSystem::HandleIntialOnlyResponse(const Worker_EntityQueryResponseOp& Op)
+{
+	USpatialNetDriver* Driver = Cast<USpatialNetDriver>(NetDriver);
```

Comment:
We want this data to be handled via the View, so the data is consistent with the rest of the entity data